### PR TITLE
Add API modification feature and respective schema for variadic features

### PR DESCRIPTION
**Pull request by jonathanKingston**
**Asana Task/Github Issue:** https://app.asana.com/0/1201048563534612/1208614628263743/f

## Description

Add support for API modification.
Also adds schema files for any variadic feature we have.

### Feature change process:

- [x] I have added a [schema](https://github.com/duckduckgo/privacy-configuration/tree/main/schema) to validate this feature change
- [x] I have tested this change locally
- [x] This code for the config change is ready to merge
- [x] This feature was covered by a tech design

#### Additional info:

- [ ] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [x] This code for the config change is ready
- [ ] This change was covered by a ship review

#### Reference

-   [Config Reviewer Documentation](https://app.asana.com/0/1200890834746050/1204443212791216/f)
-   [Config Maintainer Documentation](https://app.asana.com/0/1200890834746050/1200573250322769/f)
-   [Feature Implementer Documentation](https://app.asana.com/0/1200890834746050/1201498956177210/f)

### CODE REVIEW

File: overrides/browsers/chrome-override.json [hunk 1]
```diff
@@ -1,5 +1,60 @@
 {
     "features": {
+        "apiManipulation": {
+            "state": "enabled",
```
Comment (dharb) [inline overrides/browsers/chrome-override.json:4]: Should we do this for Edge too?

Comment (jonathanKingston) [reply overrides/browsers/chrome-override.json:4]: Yeah possibly. I think we're going to have to move this to Monday as shipping this on Friday isn't super wise.

Comment (dharb) [reply overrides/browsers/chrome-override.json:4]: Yeah good call.

**Review by dharb — APPROVED** [review]
Looks good to me 👍

**Pull request merged**